scripts/release/debian/build.sh

```diff
@@ -23,6 +23,9 @@ apt-get update
 # uuid-dev is used to build _uuid module: https://github.com/python/cpython/pull/3796
 apt-get install -y libssl-dev libffi-dev python3-dev debhelper zlib1g-dev uuid-dev
 apt-get install -y wget
+# Git is not strictly necessary, but it would allow building an experimental package
+# with dependency which is currently only available in its git repo feature branch.
+apt-get install -y git
```

Comment:
@jiasli, once MSAL 1.18.0 being released, you can go ahead to change those setup.py and requirements.py3.Linux.txt accordingly. And if you don't mind, we can keep this line here, so that our future experiments would be easier.

Reply:
Sure. I will keep this line.